src/Swift.Bindings/src/Demangler/TbdParser/TbdParser.cs

```diff
@@ -22,25 +23,25 @@ public class TbdParser
         /// <summary>
         /// Initializes a new instance of the TBD parser with a default logger
         /// </summary>
-        public TbdParser() : this(new ConsoleLogger { MinimumLevel = LogLevel.Info })
+        public TbdParser() : this(NullLoggerFactory.Instance)
```

Comment:
Nit: I would remove this constructor - it's only used in tests and in tests it's better to be explicit about where the logging will go (if anywhere). Also this means that tests don't use the same code as the product, which is always a bit iffy.